### DISCUSSION TITLE
Having issue with API

### DISCUSSION THREAD

**Discussion by s2b-git** [YetiForceCompany/YetiForceCRM#14920] [How to] [answered]
Before upgrade to 6.2 below was connecting properly. Now seems like it went out of control. His is script header (credentials fake).
Is it formatted properly?

<?php
require_once('include/main/WebUI.php');
require_once('folder1/yetirest.php');

$host = 'http://myhost.com/';
$wsname = 'user';
$wspass = 'djvpkNTeBSGsrPkg';
$wstoken = 'SyfdrZfcaudfgdhjuKkMcjUVCNmf3EG';
$user = 'user';
$pass = 'djvpkNTeBSGsrPkg';

$yr = new YetiRest($host, $wsname, $wspass, $wstoken);
$login = $yr->login($user, $pass);

**Comment by s2b-git** [comment]
`<?php
require_once('include/main/WebUI.php');
require_once('folder1/yetirest.php');

$host = 'http://myhost.com/';
$wsname = 'user';
$wspass = 'djvpkNTeBSGsrPkg';
$wstoken = 'SyfdrZfcaudfgdhjuKkMcjUVCNmf3EG';
$user = 'user';
$pass = 'djvpkNTeBSGsrPkg';

$yr = new YetiRest($host, $wsname, $wspass, $wstoken);
$login = $yr->login($user, $pass);`

**Reply by mariuszkrzaczkowski** [reply]
https://github.com/YetiForceCompany/YetiForceScripts/tree/master/YetiREST

**Comment by mariuszkrzaczkowski** [comment, marked as answer]
There were a lot of changes in api, check carefully, e.g. url address

**Reply by s2b-git** [reply]
Marius, I will get back to it in days coming, but may I ask you again, does new API (open source) have some limitations?

**Reply by waran70** [reply]
You guys  going out of mind with regressions.

**Reply by TonyM15** [reply]
... and Progression. Ubunt model will suit if here is serious approach. Lock stable for 2 years and support continuously... 
Cut out number of modules, get back to old API, keep improving. Dream?

**Reply by mariuszkrzaczkowski** [reply]
> Marius, I will get back to it in days coming, but may I ask you again, does new API (open source) have some limitations?

RestApi - no limits 
https://doc.yetiforce.com/api/?urls.primaryName=RestApi

Portal - requires a paid subscription, possibly only until the costs of building the Portal are reimbursed , The API contains all RestApi methods and adds new ones only for the needs of the portal
https://doc.yetiforce.com/api/?urls.primaryName=Portal

**Reply by mariuszkrzaczkowski** [reply]
> You guys going out of mind with regressions.


> ... and Progression. Ubunt model will suit if here is serious approach. Lock stable for 2 years and support continuously...
> Cut out number of modules, get back to old API, keep improving. Dream?

there is no wise answer to stupid questions ...
